src/main/java/com/sleekbyte/tailor/utils/CliArgumentParser.java

```diff
@@ -60,6 +60,15 @@
     private CommandLine cmd;
 
     /**
+     * Exception thrown when option parsing fails.
```

Comment:
This is currently unused. Please rename it to match (probably `CliArgumentParserException`) and remove the duplicate exception that is currently in a separate file.